lib/generateClass.js

```diff
@@ -296,16 +295,23 @@ function writeToFhir(def, fhir, cw) {
 
   const fhirProfile = [...fhir.profiles, ...fhir._noDiffProfiles].find(p => p.id === fhirID(def.identifier));
   const fhirExtension = fhir.extensions.find(p => p.id === fhirID(def.identifier, 'extension'));
+  const alreadyMappedElements = new Map;
 
   cw.ln(`let inst = {};`);
+  cw.ln();
 
   if(fhirProfile !== undefined){
     // FHIR profile exists, so we will pull out the mapping information from element maps
 
     if(def.isEntry){
+      cw.ln('// Export entry-related elements to the FHIR JSON');
       cw.ln(`inst['resourceType'] = '${fhirProfile.type}';`);
+      cw.ln(`inst['id'] = this.entryInfo.entryId.value;`);
```

Comment:
In the spirit of defensive programming, we probably should put in some `null` checks in case `this.entryInfo` or `this.entryInfo.entryId` doesn't exist.

Reply:
Yeah, that sounds reasonable. I'll fix this when I get a chance.